src/app/components/core/south/add-service-wizard/add-service-wizard.component.ts

```diff
@@ -88,6 +89,12 @@ export class AddServiceWizardComponent implements OnInit {
     }
   }
 
+  getDescription(selectedPlugin) {
+    this.isSinglePlugin = true;
```

Comment:
why is this required? in north, we  have `changedSelectedPlugin` and also set `isValidPlugin`